SgmlReaderDll/SgmlParser.cs

```diff
@@ -1503,35 +1500,35 @@ public class ElementDecl
         /// <param name="exclusions"></param>
         public ElementDecl(string name, bool sto, bool eto, ContentModel cm, string[] inclusions, string[] exclusions)
         {
-            m_name = name;
-            m_startTagOptional = sto;
-            m_endTagOptional = eto;
-            m_contentModel = cm;
+            Name = name;
+            StartTagOptional = sto;
+            EndTagOptional = eto;
+            ContentModel = cm;
             m_inclusions = inclusions;
             m_exclusions = exclusions;
         }
 
         /// <summary>
         /// The element name.
         /// </summary>
-        public string Name => m_name;
+        public string Name { get; }
 
         /// <summary>
         /// The <see cref="Sgml.ContentModel"/> of the element declaration.
         /// </summary>
-        public ContentModel ContentModel => m_contentModel;
```

Comment:
Why did you change this?  => m_contentModel is the now recommended way of exposing read only properties on a private member variable.

Reply:
thanks.